static/src/js/components/Timeline/__tests__/Timeline.test.jsx

```diff
@@ -1,22 +1,28 @@
 import React from 'react'
```

Comment:
revisit this test suite and figure out how to test component functionality in accordance to RTL best practices